saliency/base.py

```diff
@@ -95,3 +98,83 @@ def GetMask(self, x_value, feed_dict={}):
     """
     feed_dict[self.x] = [x_value]
     return self.session.run(self.gradients_node, feed_dict=feed_dict)[0]
+
+
+OUTPUT_GRADIENTS = 'OUTPUT_GRADIENTS'
+CONVOLUTION_LAYER = 'CONVOLUTION_LAYER'
```

Comment:
Is it the key for the convolutional layer values? CradCAM also requires convolutional layer gradients.

Reply:
Yes this is the values; the gradients are the next line. Do you think that CONVOLUTION_LAYER_VALUES or CONVOLUTION_LAYER_VALUE makes this more clear?

Reply:
Changed to "CONVOLUTION_LAYER_VALUES."